src/common-lib/urls/getRelativeConfigURLs.js

```diff
@@ -0,0 +1,45 @@
+// URLs should end with a `/` to avoid redirects from
```

Comment:
Is this still a problem? Given none of the URLs end in `/` maybe we can delete this comment